pkg/JuliaExperimental/ipynb/GAPJulia_Singular_car.ipynb

```diff
@@ -101,8 +101,7 @@
    ],
    "source": [
     "indetnames:= ConvertedToJulia( [ \"x0\", \"x1\", \"x2\", \"x3\" ] );\n",
```

Comment:
I think `ConvertedToJulia` is not needed, here?!

Reply:
Yes, `ConvertedToJulia` and `ConvertedFromJulia` are no longer available and are not needed.
(And also some outputs look meanwhile different from what is shown in the notebook.)

I had not looked at this notebook for a long time because of the problems with the Jupyter kernel for GAP.
Now I have tried GAP's JupyterKernel again. Without Julia connection, it seems to work (which was a surprise for me).
When I try to install JupyterKernel for a GAP that is running under Julia then Julia crashes when GAP loads ZeroMQInterface.

Reply:
AFAIK the GAP JupyterKernel uses `fork`, so I am surprised it doesn't crash earlier (it definitely used to).

On the GAP slack, just this week a student from St Andrews turned up who reported that he has been hired to work on the JupyterKernel package. So here's hoping...